.rubocop.yml

```diff
@@ -61,7 +62,7 @@ Style/FileName:
 Style/HashSyntax:
   Severity: fatal
   Enabled: true
-  EnforcedStyle: hash_rockets
+  EnforcedStyle: ruby19_no_mixed_keys
```

Comment:
I think the reasoning was this was a bit more difficult to read at a glance...

```rb
d = {foo: :bar, baz: :qux}
# => {:foo=>:bar, :baz=>:qux}
```

...and as you can see it's just syntastical sugar, as Ruby returns hash rockets any way